alliance.avg.rankings.user.js

```diff
@@ -1,14 +1,13 @@
 // ==UserScript==
 // @name         Dark Galaxy - Alliance average rankings
 // @namespace    https://darkgalaxy.com/
-// @version      0.5
+// @version      0.6
 // @description  All your planet are belong to us
 // @author       Biggy
 // @homepage     https://github.com/fl0v/dg
 // @supportURL   https://github.com/fl0v/dg/issues
 // @downloadURL  https://github.com/fl0v/dg/raw/master/alliance.avg.rankings.user.js
-// @match        https://beta.darkgalaxy.com/rankings/alliances/
-// @match        https://andromeda.darkgalaxy.com/rankings/alliances/
+// @match        https://*.darkgalaxy.com/rankings/alliances/
```

Comment:
i dont remember exactly why i chose to use the exact domains but if you tested  * as subdomain and it works then i guess its fine